plugins/woocommerce/client/blocks/assets/js/blocks/checkout/inner-blocks/checkout-terms-block/frontend.tsx

```
@@ -90,24 +90,20 @@ const FrontendBlock = ( {
-						</CheckboxControl>
-					</>
+					<CheckboxControl
+						id="terms-and-conditions"
+						checked={ checked }
+						onChange={ () => setChecked( ( value ) => ! value ) }
+						hasError={ hasError }
+						disabled={ isDisabled }
+					>
+						<span
+							className="wc-block-components-checkbox__label"
+							dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)